incubator/exporter-otlp/src/main/java/io/aklivity/zilla/runtime/exporter/otlp/internal/config/OtlpEndpointConfig.java

```diff
@@ -12,21 +12,23 @@
  * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
  * specific language governing permissions and limitations under the License.
  */
-package io.aklivity.zilla.runtime.exporter.prometheus.internal.layout;
+package io.aklivity.zilla.runtime.exporter.otlp.internal.config;
 
-public abstract class Layout implements AutoCloseable
-{
-    public enum Mode
-    {
-        CREATE_READ_WRITE,
-        READ_ONLY
-    }
+import io.aklivity.zilla.runtime.engine.config.OptionsConfig;
 
-    @Override
-    public abstract void close();
+public class OtlpEndpointConfig extends OptionsConfig
+{
+    public String protocol;
+    public String location;
```

Comment:
This should be of type `URI` instead of `String`.

Suggest adding a `URI resolveMetrics()` method returning `location.resolve(overrides.metrics)`.
Similarly for `resolveTraces()` and `resolveLogs()`.

Reply:
thx, fixed